visualize/grpc/pb/visualize_grpc.pb.go

```diff
@@ -3347,6 +3493,24 @@ func _VisualizeGrpcService_CtrlSetTitle_Handler(srv interface{}, ctx context.Con
 	return interceptor(ctx, in, info, handler)
 }
 
+func _VisualizeGrpcService_Command_Handler(srv interface{}, ctx context.Context, dec func(interface{}) error, interceptor grpc.UnaryServerInterceptor) (interface{}, error) {
+	in := new(CommandRequest)
+	if err := dec(in); err != nil {
+		return nil, err
+	}
+	if interceptor == nil {
+		return srv.(VisualizeGrpcServiceServer).Command(ctx, in)
+	}
+	info := &grpc.UnaryServerInfo{
+		Server:     srv,
+		FullMethod: "/visualize_grpc_pb.VisualizeGrpcService/Command",
```

Comment:
ditto.

Reply:
It's auto-generated. Please report the issue to [protoc-gen-go](https://github.com/golang/protobuf) developers.